ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/432__data-science-pipelines-tekton.robot

```diff
@@ -34,10 +34,10 @@ End To End Pipeline Workflow Using Kfp Tekton
     New Project    ${project}
     Install DataSciencePipelinesApplication CR    ${project}
```

Comment:
is this still necessary in 1.27?

Reply:
DataSciencePipelinesApplication (DSPA) is per project